src/viz-sentence-population/VizSentencePopulation.js

```diff
@@ -0,0 +1,175 @@
+import PropTypes from "prop-types";
```

Comment:
FYI, all of the duplication in this file disappears in the [next commit](https://github.com/Recidiviz/public-dashboard/pull/71/commits/6157461d056572263886b3caa33d9d5bfc2a780e)